includes/actions/RevertAction.php

```diff
@@ -113,6 +113,9 @@ public function onSubmit( $data ) {
 	}
 
 	public function onSuccess() {
+		// Wikia change - Add hook for a successful file revert
+		wfRunHooks( 'FileRevertComplete', [ $this->page ] );
```

Comment:
No we don't have a convention for this. I do think `Hooks::run` is better, or rather that `wfRunHooks` is redundant, but kept with `wfRunHooks` here to maintain consistency with other core files in our version of MW. I don't have a strong attachment to doing so though. ;)